labgrid/resource/ykushpowerport.py

```diff
@@ -0,0 +1,17 @@
+import attr
+
+from ..factory import target_factory
+from .common import Resource
+
+
+@target_factory.reg_resource
+@attr.s(cmp=False)
+class YKUSHPowerPort(Resource):
+    """This resource describes a YEPKIT YKUSH switchable USB hub.
+
+    Args:
+        serial (str): serial of the YKUSH device 
+        index (str): port index"""
+    serial = attr.ib(validator=attr.validators.instance_of(str))
+    index = attr.ib(validator=attr.validators.instance_of(str),
+                    convert=lambda x: str(int(x)))
```

Comment:
If you use ``convert=lambda x: int(x)``, you should be able to remove the int() conversions in the on/off functions.